src/scanoss/spdxlite.py

```
@@ -30,6 +30,7 @@
 import getpass
 import re
 import importlib_resources
+from commoncode.fetch import download_url
```

Ruff (0.8.2)

`commoncode.fetch.download_url` imported but unused

Remove unused import: `commoncode.fetch.download_url`

(F401)